src/main/java/io/dashboardhub/pipelinedashboard/domain/User.java

```diff
@@ -23,10 +26,10 @@
     private String username;
 
     private String name;
+    private String email;
 
-//    @ManyToMany
-//    @JoinTable(name="users_projects")
-//    private Set<Project> projects;
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
this is gonna kill performance if there is a login recorded each time the user logs in. This set will become massive and be eagerly fetched each time the User record is. Suggest reversing the relationship as knowledge of login history only really valid in admin/support/debug situations.